spring-vault/pom.xml

```diff
@@ -38,7 +54,7 @@
     </dependencies>
 
     <properties>
-        <spring.vault.core.version>2.1.1.RELEASE</spring.vault.core.version>
+        <spring.vault.core.version>2.2.0.RELEASE</spring.vault.core.version>
```

Comment:
Could we try with version 2.2.3.RELEASE ?

Reply:
done